docs/topics/proc_add-debug-logging-nodejs.adoc

```diff
@@ -0,0 +1,36 @@
+= Add Debug Logging
+There are several options for debug logging. This example uses the link:https://www.npmjs.com/package/debug[debug package].
+
+
+.Prerequisites 
+* An application you want to debug. For example, xref:mission-http-api-nodejs[a booster].
+
+.Procedure
+
+. Add the `debug` logging definition.
++
+[source,js,options="nowrap",subs="attributes+"]
+----
+var debug = require('debug')('mybooster');
+----
+
+. Add debug statements.
++
+[source,js,options="nowrap",subs="attributes+"]
+----
+app.use('/api/greeting', (request, response) => {
+  const name = request.query ? request.query.name : undefined;
+  //log name in debugging
+  debug('name: '+name);
+  response.send({content: `Hello, ${name || 'World'}`});
+});
+----
+
+. Install the link:https://www.npmjs.com/package/debug[debug] module.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ npm install debug
+----
++
+Depending on your application, this module may already be included. For example, link:https://expressjs.com[express] applications link:https://expressjs.com/en/guide/debugging.html[already include this module], therefore you do not need to install it.
```

Comment:
I'm not sure I would include this text. It puts the burden on the user to figure out which packages include it and which don't. And even if some do, it will not hurt to install it.